modules/ROOT/pages/troubleshooting.adoc

```diff
@@ -121,27 +121,78 @@ In this release, IP restrictions for the connected app are not supported. By def
 
 An error occurs when saving styling changes to the portal if an object has validation rules applied for standard objects that are created during the Salesforce linking process. 
 
-The workaround is to deactivate the validation rule, save the styling changes, then reactive the validation rule: 
+Validation rules cannot be applied to Salesforce standard objects, for example, account and user objects, that are created during the Salesforce linking process. 
 
-. Navigate to *Setup* > *Object Manager*.
-. Select *Validation Rules*.
-. Search for the object with the validation rule.
-. Click *Edit* next to the rule you want to deactivate.
-. Deselect the *Active* checkbox.
-. Click *Save*.
-. Navigate back to API Experience Hub and click *Try again* in the error message.
-. Click *Save* to save your style selections.
-. Repeat steps 1 through 4.
-. Select the *Active* checkbox and click *Save*.
+The workaround is to bypass the validation rule for the Salesforce user who performs the linking process. 
 
+NOTE: These instructions pertain to bypassing account object validation rules but can apply to bypassing custom validation rules of other objects. 
+
+. Create a custom permission:
+.. From Salesforce, go to *Setup*, enter *custom permissions* in the *Quick Find* box, and select *Custom Permissions*.
+.. Click *New* and enter the permission information:
++
+* *Label*
++ 
+Enter *Bypass Account Validation Rule*. 
++
+* *Name* 
++
+Click the field to populate it with the label name with underscores.
++
+* *Description*
++ 
+Enter a description for the permission.
+.. Click *Save*.
+
+. Create a permission set and add the custom permission:
```

Comment:
It's a little weird to have the colon introducing the sub-list broken up by the hanging paragraph. Is there another place you can put the following sentence?